engine/lib/phx/src/ui/hmgui/widget.rs

```diff
@@ -251,97 +269,64 @@ impl HmGuiWidget {
-        println!("{ident_str}{IDENT}- horiz_align:    {:?}", self.vertical_alignment);
-        println!("{ident_str}{IDENT}- vert_align:     {:?}", self.horizontal_alignment);
-        println!("{ident_str}{IDENT}- margin_upper:   {:?}", self.margin_upper);
-        println!("{ident_str}{IDENT}- margin_lower:   {:?}", self.margin_lower);
-        println!("{ident_str}{IDENT}- bg_color:       {:?}", self.bg_color);
-        println!("{ident_str}{IDENT}- border_width:   {}",   self.border_width);
-        println!("{ident_str}{IDENT}- border_color:   {:?}", self.border_color);
-        println!("{ident_str}{IDENT}- min_size:       {:?}", self.min_size);
-        println!("{ident_str}{IDENT}- inner_min_size: {:?}", self.inner_min_size);
-        println!("{ident_str}{IDENT}- hash:           0x{:X?}", self.hash);
-        println!("{ident_str}{IDENT}- render_style:   {:?}", self.render_style);
-        println!("{ident_str}{IDENT}- mouse_over:     {:?}", self.mouse_over);
+        println!("{ident_str}{IDENT}- pos:              {:?}", self.pos);
+        println!("{ident_str}{IDENT}- size:             {:?}", self.size);
+        println!("{ident_str}{IDENT}- inner_pos:        {:?}", self.inner_pos);
+        println!("{ident_str}{IDENT}- inner_size:       {:?}", self.inner_size);
+        println!("{ident_str}{IDENT}- default_width:    {:?}", self.default_width);
+        println!("{ident_str}{IDENT}- default_height:   {:?}", self.default_height);
+        println!("{ident_str}{IDENT}- horiz_align:      {:?}", self.vertical_alignment);
+        println!("{ident_str}{IDENT}- vert_align:       {:?}", self.horizontal_alignment);
+        println!("{ident_str}{IDENT}- margin_upper:     {:?}", self.margin_upper);
+        println!("{ident_str}{IDENT}- margin_lower:     {:?}", self.margin_lower);
+        println!("{ident_str}{IDENT}- border_width:     {}",   self.border_width);
+        println!("{ident_str}{IDENT}- background_color: {:?}", self.background_color);
+        println!("{ident_str}{IDENT}- highlight_color:  {:?}", self.highlight_color);
+        println!("{ident_str}{IDENT}- opacity:          {}", self.opacity);
+        println!("{ident_str}{IDENT}- min_size:         {:?}", self.min_size);
+        println!("{ident_str}{IDENT}- inner_min_size:   {:?}", self.inner_min_size);
+        println!("{ident_str}{IDENT}- hash:             0x{:X?}", self.hash);
+        println!("{ident_str}{IDENT}- mouse_over:       {:?}", self.mouse_over);
         println!("{ident_str}{IDENT}# item: {}", self.item.name());
 
         match &self.item {
             WidgetItem::Container(item) => item.dump(ident + 1),
             WidgetItem::Text(item) => item.dump(ident + 1),
-            WidgetItem::Rect(item) => item.dump(ident + 1),
+            WidgetItem::Rect => {
+                let ident_str = format!("{}", crate::ui::hmgui::IDENT.repeat(ident+1));
+
+                println!("{ident_str}- rect");
```

Comment:
intended?

Reply:
Yes, originally it was in a separate rect method but I removed it and put print here.